<launch>
<arg name = "pathToRosbag"  default = "/home/dev/percept/sample-data/d435i_sample_data/d435i_walking.bag" />


<group>
	<node pkg="percept" type="real_time_pipeline.py" name="perception_node" output="screen">
		<param name="log_level" value="debug"/>
	</node>
	<!-- <node pkg="percept" type="message_saver.py" name="message_saver" output="screen">
		<param name="log_level" value="debug"/>
	</node> -->
</group>

<group>
	<arg 	name = "selectedTopics"  
				default = "--topics \
				/device_0/sensor_0/Depth_0/image/data \
				/device_0/sensor_0/Depth_0/image/metadata \
				/device_0/sensor_0/Depth_0/info \
             	/device_0/sensor_0/Depth_0/info/camera_info \
             	/device_0/sensor_0/Depth_0/tf/0" />

	<node pkg  = "rosbag"
		type = "play"
		name = "rosbag_player"
		
				args = "$(arg pathToRosbag) $(arg selectedTopics) -l"/>
</group>



</launch>

